packages/flutter/test/widgets/draggable_scrollable_sheet_test.dart

```diff
@@ -1892,4 +1892,29 @@ void main() {
     expect(receivedNotification!.shouldCloseOnMinExtent, isFalse);
     controller.dispose();
   });
+
+  testWidgets('Ensures DraggableScrollableSheet snaps exactly to minChildSize', (
```

Comment:
Can you add a link to the issue in a comment? `Regression test for https://github.com/flutter/flutter/issues/140701.`